<!DOCTYPE html>
<html>
<body>
  <h1>Protein Information</h1>
  <p>
    All proteins are made from the same 20 amino acids, in different combinations and lengths. Amino acids can be categorized into groups based on side chain properties. They may be polar, nonpolar, acidic or basic. Proteins have four levels of structural organization. The primary structure corresponds to the amino acid sequence. The secondary structure is determined by the orientation and interactions of amino acid side chains also called R-groups. Secondary structure includes beta sheets, various helices and coils. The tertiary protein structure categorizes the arrangements of secondary structural elements. The quaternary structure describes the overall shape and arrangement of proteins with multiple chains. Proteins fold to minimize the free energy of the system. Hydrophobic collapse acts as one of the main forces that guid protein folding.<br>
Proteins serve a variety of essential cellular functions. They may have a globular or fibrous structure. Globular proteins appear as more of a mass with a hydrophilic solvent accessible surface and a hydrophobic core. Globular proteins may act as catalytic enzymes, transporters and regulatory or messenger molecules. Fibrous proteins serve mainly as a structural support system. Globular proteins often interact to form more complex systems. G-protein coupled receptors contain a transmembrane domain made of seven alpha helices, and an intracellular component. The intracellular component has an alpha, beta and gamma subunit which have additional interaction and when activated propagate a secondary messenger system.<br>
A common theme in biology is a relationship between structure and function, proteins follow this rule. Proteins often bind metal cofactors, or undergo a variety of post-translational modifications. These modifications often result in structural changes that produce a protein in the active state. Primary structural characteristics include size and shape. Hydrophobicity is another structural characteristic determine function; it be deduced from the primary and secondary sequence. The cell membrane, a phospholipid bilayer, internally consists of fatty chains. Transmembrane proteins contain alpha helices, which contain many hydrophobic amino acids. The alpha helices interact favorably with fatty chains inside the membrane. The arrangement of distant amino acids is usually essential for catalytic function. Interactions of distant side chain may form the active site for a ligand. The binding of the ligand to the active site triggers a conformational shift to a lower energy transition state. The transition state facilitates the chemistry of the reaction to occur. Through evolution, many different proteins have converged and diverged. Modern technology enables protein sequencing and the storage of large amounts of data. <br>
Through this primary sequence data, regions of proteins have been categorized. Patterns of amino acids often correspond to a specific type of structure. Domains are 3-dimensional, tertiary, structures that may function, evolve and exist separately from the rest of a protein. Computational tools exist to search protein sequences for known domain patterns. Motifs are smaller amino acid patterns. There are structural and sequence motifs. Sequence motifs are widespread and can confer some biological activity, usually corresponding to subcellular localization. Structural motifs can be thought of as a super-secondary structure. They contain a few secondary structures that have some function. The Zinc Finger motifs consist of an alpha helix and two anti-parallel beta sheets. The shape resembles the thumb almost touching the index and middle finger. At this intersection a metal, like Zinc may bind. Zinc finger motifs are an indicator that a protein binds to DNA.8
Protein evolution has created enormous diversity. Since domains may exist and function independently, they serve as great instruments for protein evolution. There are a few basic mechanisms through which they evolve. Domains may be lost, transferred, duplicated or combined. These mechanisms can change the overall shape, catalytic sites, binding interactions and subcellular localization. Some proteins exist as homodimers, in which a duplication likely resulted in a more complex structure. The combinations of domains is another very important evolutionary mechanism. Domains are often loosely connected by some kind of coil structure. At the site where they interact, they can form new binding surfaces for different ligands and create new functions. Evolution may be convergent or divergent. In convergent evolution, unrelated proteins develop similar functions. In divergent evolution, related proteins slowly change and develop new, different functions. <br>
The modern health care system in the United States is extensive and pharmaceutical companies are regularly developing drugs to cure diseases. The drugs undergo rigorous trials in which large groups of people are analyzed for reactions to the medications. Generally, drugs interact with proteins to have a regulatory effect on pathways in the body. Non-steroidal anti-inflammatory drugs binds to cyclooxygenase-2 (COX 2) proteins to down regulate inflammation. The common NSAIDS non-selectively bind to both COX-2 and COX-1. COX-1 helps to maintain the lining of the intestines and regulate blood clotting. The effect of drugs on systems is not fully understood. The ligand that naturally binds to COX, arachidonic acid, binds to a series of proteins in a pathway to effect changes. The overall system is impacted on a level greater.<br>
Drugs
ligands to enzymes
Access metabolic systems
	Systems biology uses a combination of computational scientific tools to study interactions within biological systems. Applying systems biology to analyze metabolic pathways enables a more complete understanding of the underlying processes surrounding the large scale processes, for example carbohydrate metabolism and specific contributing mechanisms. Natural ligands in metabolic pathways bind to a series of enzymes and undergo reactions at various points in the path to become a useful molecule. When the ligand that bound to one enzyme in the path is released after a reaction, the metabolite maintains a relatively similar structure. The similarly structured metabolite becomes a ligand for the next enzyme in the pathway. SInce the original ligand and metabolite have shared structural characteristics, the binding region of the enzymes may also contain some similarities. The enzymes may not have a clear homology relationship and a sequence based comparison may not indicate a similar binding region. The 3D geometry of the ligand in the binding regions of enzymes in a pathway may contain some similarities. Combining systems biology with the structural proteome is actively being researched. The goal of that research is to map genes to functional protein products. Experimentally determined structures are gathered from databases while the unknown proteins are modelled.<br>
Natural ligands bind to enzymes in pathways
Ligands and metabolites have similar structures
Protein active site amino acids theoretically contain similar binding interactions with the ligands
Also may have similar 3D geometry at the active site
Intersection of systems biology and structural proteome
GEM/PRO
	Metabolic pathways are important for many biological functions. The central metabolism of Escherichia Coli is very complex and


  </p>
<a href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/basic.html">Return to Tutorial</a> <br>

</body>
</html>
