<!DOCTYPE html>
<html>
<body>

<body style="background-color:powderblue;">
<h1> <center> Advanced Bioinformatics Project</h1>
<h2> <center> Tool Name: Pit Viper</h2>
<center> <img src="viper.jpg" alt="Pit Viper" width="200" height="150"> </center>

<p>
  This project will serve as a tutorial to analyze proteins in
  a related pathway that operate with similar ligands. Chimera
  and PDB are the primary references that will be used. Links are
  provided in the sections below.
</p>

<h1> <center> Background Information: Proteins and Systems </center> </h1>
<p>
  <a href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/background.html">Background Information</a> <br>
</p>

<h1> Step By Step Walkthrough</h1>
<p>
  <strong>Step One: Use PDB pathway tool </strong><br>
  The PDB archive contains information about experimentally-determined structures <br>
  of proteins, nucleic acids, and complex assemblies.<br>
  The pathway tool specifically outlines metabolic systems and indentifies the proteins and ligands.<br>
  Any additonal information regarding PDB can be found on the website linked below. <br>
  Some releveant information is included in the background section.<br>
    <a
    href="https://www.rcsb.org/pdb/pathway/pw.do"> PDB Pathway View
    </a> <br>
    <a href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step1.html">Step One
    </a> <br>
</p>
<p>
  <strong>Step Two: Using the Map to Isolate Proteins and Ligand </strong><br>
  The PDB pathway viewer provides ligands and proteins in coordinated pathways. <br>
  Using the path view, proteins with known 3D structures are identifiable with blue arrows. <br>
  Ligands with structures documented in PDB are represented as blue dots. <br>
  Proteins without PDB structres are represented by yellow lines and undocumented ligands are grey dots. <br>
  There is an option to show the full names of the protein, the option is under map in the pathway view search. <br>
  Another useful database for examining pathways is BRENDA, the link is provided below. <br>
  BRENDA may provide additional information regarding the enzymes and ligands.<br>
  <a
  href="https://www.brenda-enzymes.org"> BRENDA - Comprehensive Enzyme Information System
  </a> <br>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step2.html">Step Two
  </a> <br>
</p>

<p>
  <strong>Step Three: Use PDB to find 3D protein structures </strong><br>
  PDB provides various tools and parameters to search for proteins in the database. <br>
  The advanced search provides a basic quick search and additional searches based upon:<br>
  ID(s) and Keywords <br>
  Structure Annotation<br>
  Deposition<br>
  Structure Features<br>
  Sequence Features<br>
  Chemical Components<br>
  Biologically Interesting Molecules<br>
  Biology<br>
  Methods<br>
  Publication<br>
  Miscellaneous<br>
  The search results contain filters that can narrow the results. <br>
  The filters enable the user to find the desired protein with greater specificity. <br>
    <a
    href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step3.html">Step Three
    </a> <br>
</p>
<p>
<strong>Step Four: Identify Important Amino Acids</strong> <br>
PDB provides information regarding the macromolecule (protein) and small molecules. <br>
Small molecule information details the chain(s) the ligand is bound to, <br>
the ligand molecular formula, a 2D diagram (some entries also have a 2D interaction summary)<br>
and 3D interaction information. The 3D ligand interaction provides an analysis of the ligand bound <br>
to critical amino acids. The information should be recorded for steps seven and eight. <br>
There is also a link to uniprot within the macromolecule section. <br>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step4.html">Step Four
  </a> <br>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step4UNI.html">5UZP Uniprot Information
  </a> <br>
</p>
<p>
<strong>Step Five: Open Structures in Chimera </strong><br>
UCSF Chimera is a highly extensible program for interactive visualization and analysis of molecular structures <br>
and related data, including density maps, supramolecular assemblies, sequence alignments, docking results, <br>
trajectories, and conformational ensembles. High-quality images and animations can be generated. Chimera <br>
includes complete documentation and several tutorials, and can be downloaded free of charge for academic, <br>
government, nonprofit, and personal use. Chimera is developed by the Resource for Biocomputing, <br>
Visualization, and Informatics (RBVI), supported in part by the National Institutes of Health (NIGMS P41-GM103311). <br>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step5.html">Step Five
  </a> <br>
Provided below are links to Chimera commands and the UCSF tutorials<br>
<a href="https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/frametut.html">UCSF Chimera Complete Tutorial</a> <br>
<a href="https://www.cgl.ucsf.edu/chimera/docs/UsersGuide/quickref.pdf">Chimera Quick Refernce Sheet</a> <br>
</p>
<p>
<strong>Step Six: Identify Ligand Infortmation </strong><br>
Chimera serves many functions. The user may work directly with the command line or may make .cmd scripts using a text editor. <br>
In Step X, the text editor Atom was used to generate scripts based on information <br>
gathered throughout the other steps. In Chimera, the user isolates the ligand and <br>
amino acids critical for binding. Many features may be turned on or off to examine <br>
specific aspects of the structure. In this example the solvent, ribbon and excess chains are removed. <br>
<a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step6.html">Step Six
  </a> <br>
</p>

<p>
<strong>Step Seven: Examine The Protein Structres </strong><br>
Chimera provides many tools to analyze proteins and ligands.<br>
In step seven, the protein is examined using interactive preset designs,<br>
residue labels, surface views and hydrogen bonding.<br>
The proteins are observed seperately and then compared in the next step.<br>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step7.html">Step Seven
  </a> <br>
</p>

<p>
<strong>Step Eight: Superpose the Structures</strong><br>
A couple useful Chimera tools are the match and matchmaker functions. <br>
Match enables the user to superpose strucutres based on a set <br>
of at least three distinct atoms. For the example, atoms are provided. <br>
The matchmaker function performs a sequence alignment of protein strucutres <br>
to determine if the proteins can be superposed. It is more than just a <br>
basic sequence alignment, 3D structural characterisitcs are factored in.<br>
The superposition enables a more direct comparison of the ligand bound in the <br>
different active sites.<br>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step8.html">Step Eight
  </a> <br>
</p>

<p>
<strong>Step Nine: Additional Analysis</strong><br>
Proteins have many important distinguishing characterisitics. <br>
The internet is host to many different tools to analyze proteins. <br>
Uniprot and BRENDA are two other resoures that provide supplemental information. <br>
The mission of UniProt is to provide the scientific community with a comprehensive, <br>
high-quality and freely accessible resource of protein sequence and functional information.  <br>
Through BRENDA a fold classification analysis reveals strucutural charecteristics of the protein. <br>
The aresenal of BLAST algorithms also provide insight into the evolution and classification <br>
of the protein in question. <br>

  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/step9.html">Step Nine
  </a> <br>
</p>

<p>
<strong>Step X: Generate Command Script </strong><br>
<a
href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/stepx.html">Step X
</a> <br>
<br>
</p>

<h1>Additional Examples</h1>
<p>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/example2/example2.html">Example Two
  </a> <br>
  <a
  href="file:///Users/jamesbuckley/Google Drive/college-classes/senior/semester_2/Advanced_Bioinformatics/website/tutorial/example3.html">Example Three
  </a> <br></p>

<h1>Conclusion</h1>
<p>
  The analyses of proteins in a biological system has ramifications<br>
  to examine binding relationships. There is potetnial for protein design,<br>
  drug targeting and more.<br>
</p>

<h1>Important Links</h1>
<a href="https://www.cgl.ucsf.edu/chimera/download.html">Chimera Download</a> <br>
<a href="https://www.rcsb.org/structure/5uzp">Sample PDB entry</a> <br>
<a href="https://www.rcsb.org"> PDB Website </a> <br>
</body>
</html>
